src/webapp/js/components/header.js

```diff
@@ -1,54 +1,38 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { connect } from 'react-redux';
 
-import { signoutUser } from '../actions';
+const Header = ({ signoutUser, toggleShowMenuSmart, showMenuSmart }) => (
+  <div>
+    <div className="navbar-header">
+      <button type="button" className="navbar-toggle" onClick={toggleShowMenuSmart}>
+        <span className="sr-only">Toggle navigation</span>
+        <span className="icon-bar" />
+        <span className="icon-bar" />
+        <span className="icon-bar" />
+      </button>
+      <Link className="navbar-brand" to="/" onClick={toggleShowMenuSmart}>MOTS</Link>
+    </div>
+    <ul className="nav navbar-right top-nav hide-max-r-xsmall-max">
+      <li>
+        <Link to="/" onClick={showMenuSmart ? toggleShowMenuSmart : () => {}}>
```

Comment:
why only here the sate is checked?
I think only the menu button needs to actually toggle the state, rest can just hide it, right?
can we have a new function hideMenuSmart instead and also use it here and in all other links

Reply:
I think i missed it, I made it because earlier it was outside of menu, and I didn't want it to open menu in any time, but as you say hideMenuSmart may be a better solution.